API.md

```diff
@@ -152,3 +152,63 @@ The payload should be a classifier object, with a classifier type selector `name
 An `HTTP 404 Not Found` status code is returned if the survey with the specified ID could not be found.
 
 An `HTTP 409 Conflict` status code is returned if a classifier type selector already exists for any of the names in the payload.
+
+## Post New Survey
+* `POST /surveys` will create a new survey.
+
+The payload should be a JSON document, with an `ID`, a `ShortName`, a `LongName`, a `Reference`, a `LegalBasis`, a `SurveyType` as strings, and `Classifiers` as a list.
+
+### Example JSON payload
+```json
+{
+    "ID": "efa868fb-fb80-44c7-9f33-d6800a17c4da",
```

Comment:
these keys are incorrect. with a Golang struct the bit after `json:` is the key name
`ID            string json:"id"`
so for example this should be `id` not `ID`